### PR TITLE
lang: adding translations for Catalan

### PR DESCRIPTION
**Pull request by Payetus**
Adding transalations for Catalan:

- Participans Pane:
- Toolbar 
- Video Thmbnail

### CODE REVIEW

File: lang/main-ca.json [hunk 1]
```diff
@@ -464,6 +464,32 @@
         "newDeviceAudioTitle": "S'ha detectat un aparell d'àudio nou",
         "newDeviceAction": "Usa"
     },
+"participantsPane": {
```
Comment (saghul) [inline lang/main-ca.json:467]: Can you fix the indentation here please?

Comment (Payetus) [reply lang/main-ca.json:467]: Sure

**Comment by Payetus** [comment]
@saghul Done

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Landed, cheers!

**Pull request merged**